.github/workflows/ci.yml

```diff
@@ -74,6 +75,7 @@ jobs:
           [ $(nix config show substituters) == "https://cache.nixos.org/" ]
 
   run-systems:
+    if: github.event_name != 'pull_request'
```

Comment:
this has not good vibes. Rec: combine with: `if: ${{ github.event_name == 'merge_group' }}`; ref: https://stackoverflow.com/questions/76655935/when-does-a-github-workflow-trigger-for-merge-group-and-is-it-restricted-by-bran

Reply:
```suggestion
    if: github.event_name == 'merge_group'
```